Packs/FireEyeNX/Integrations/FireEyeNX/FireEyeNX.yml

```diff
@@ -1266,5 +1236,5 @@ script:
 tests:
 - FireEyeNX-Test
 fromversion: 5.0.0
-defaultclassifier: FireEye NX - Classifier
+defaultclassifier: FireEye NX
```

Comment:
Is this correct?

Reply:
Yes, the new validation caught it, `FireEye NX - Classifier` is the name and `FireEye NX` is the id.
we need the id there